sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/rx/FullTextIndexTest.java

```diff
@@ -0,0 +1,154 @@
+
+import com.azure.cosmos.ConsistencyLevel;
+import com.azure.cosmos.CosmosAsyncClient;
+import com.azure.cosmos.CosmosAsyncContainer;
+import com.azure.cosmos.CosmosAsyncDatabase;
+import com.azure.cosmos.CosmosClientBuilder;
+import com.azure.cosmos.CosmosDatabaseForTest;
+import com.azure.cosmos.CosmosException;
+import com.azure.cosmos.DirectConnectionConfig;
+import com.azure.cosmos.implementation.TestConfigurations;
+import com.azure.cosmos.implementation.Utils;
+import com.azure.cosmos.implementation.guava25.collect.ImmutableList;
+import com.azure.cosmos.models.CosmosContainerProperties;
+import com.azure.cosmos.models.CosmosFullTextIndex;
+import com.azure.cosmos.models.CosmosFullTextPath;
+import com.azure.cosmos.models.CosmosFullTextPolicy;
+import com.azure.cosmos.models.ExcludedPath;
+import com.azure.cosmos.models.IncludedPath;
+import com.azure.cosmos.models.IndexingMode;
+import com.azure.cosmos.models.IndexingPolicy;
+import com.azure.cosmos.models.PartitionKeyDefinition;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Ignore;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
+
+@Ignore("TODO: Ignore these test cases until the public emulator with vector indexes is released.")
+public class FullTextIndexTest extends TestSuiteBase{
+    protected static final int TIMEOUT = 30000;
+    protected static final int SETUP_TIMEOUT = 20000;
+    protected static final int SHUTDOWN_TIMEOUT = 20000;
+
+    protected static Logger logger = LoggerFactory.getLogger(VectorIndexTest.class.getSimpleName());
+    private final ObjectMapper simpleObjectMapper = Utils.getSimpleObjectMapper();
+    private final String databaseId = CosmosDatabaseForTest.generateId();
+    private CosmosAsyncClient client;
+    private CosmosAsyncDatabase database;
+
+    @BeforeClass(groups = {"emulator"}, timeOut = SETUP_TIMEOUT)
+    public void before_VectorIndexTest() {
+        // set up the client
+        client = new CosmosClientBuilder()
+            .endpoint(TestConfigurations.HOST)
+            .key(TestConfigurations.MASTER_KEY)
+            .directMode(DirectConnectionConfig.getDefaultConfig())
+            .consistencyLevel(ConsistencyLevel.SESSION)
+            .contentResponseOnWriteEnabled(true)
+            .buildAsyncClient();
+
+        database = createDatabase(client, databaseId);
+    }
+
+    @AfterClass(groups = {"emulator"}, timeOut = SHUTDOWN_TIMEOUT, alwaysRun = true)
+    public void afterClass() {
+        safeDeleteDatabase(database);
+        safeClose(client);
+    }
+
+    @Test(groups = {"emulator"}, timeOut = TIMEOUT*10000)
```

Comment:
30_000 * 10_000 - that seems like a very large timeout.

Reply:
Removed the 10000